Source/JavaScriptCore/API/JSStringRefCF.cpp

```diff
@@ -37,16 +37,14 @@ JSStringRef JSStringCreateWithCFString(CFStringRef string)
 {
     JSC::initialize();
 
-    // We cannot use CFIndex here since CFStringGetLength can return values larger than
-    // it can hold.  (<rdar://problem/6806478>)
-    size_t length = CFStringGetLength(string);
+    CFIndex length = CFStringGetLength(string);
```

Comment:
But still CFIndex is 32bit, right? So does anything change?

Reply:
It's a long. Not an int. Longs are 64 bit

Reply:
Sounds good.